### PR TITLE
Add new atcommand *reloadnpc

### PR DESCRIPTION
**Pull request by Emistry**
### Pull Request Prelude

- [x] I have followed [proper Hercules code styling][code].
- [x] I have read and understood the [contribution guidelines][cont] before making this PR.
- [x] I am aware that this PR may be closed if the above-mentioned criteria are not fulfilled.

### Changes Proposed
- `@reloadnpc <path>` - unloads all NPCs in a file and reload it again.
- Be aware that mapflags and monster spawned directly are not removed.

**Issues addressed:** 
none

[cont]: https://github.com/HerculesWS/Hercules/blob/master/CONTRIBUTING.md
[code]: https://github.com/HerculesWS/Hercules/wiki/Coding-Style

### CODE REVIEW

**Comment by HerculesWSAPI** [comment]
This change is [Reviewable](https://reviewable.io/reviews/herculesws/hercules/2476)

File: src/map/atcommand.c [hunk 1]
```diff
@@ -4454,6 +4454,38 @@ ACMD(unloadnpc)
...
+{
+	if (!*message) {
+		clif->message(fd, msg_fd(fd, 1385)); // Usage: @unloadnpcfile <file name>
+		return false;
+	} else if (npc->unloadfile(message) == true) {
+		clif->message(fd, msg_fd(fd, 1386)); // File unloaded. Be aware that mapflags and monsters spawned directly are not removed.
+
+		FILE *fp;
+		// check if script file exists
+		if ((fp = fopen(message, "r")) == NULL) {
```
Comment (4144) [inline src/map/atcommand.c]: code ``fp = fopen(message, "r")`` can be moved into line with ``FILE *fp`` as ``FILE *fp = fopen(message, "r")``

Comment (Emistry) [reply src/map/atcommand.c]: done

**Comment by M45T3Ryu** [comment]
This probably could be merged with @unloadnpcfile and @loadnpcfile maybe? something like..


@npcfile load npc/lols/fasdfas.txt or @npcfile 1 npc/lols/fasdfas.txt
@npcfile reload npc/lols/fasdfas.txt  or @npcfile 2 npc/lols/fasdfas.txt
@npcfile unload npc/lols/fasdfas.txt or @npcfile 3 npc/lols/fasdfas.txt

prob a single command with arguments, I dont think this command is used on anything so we should be allowed to change them 

maybe with constants or maybe just @file making it possible to do more things with it? idk why 3 commands but i guess thats how things are?

**Pull request merged**